tests/v2/validation/rbac/rbac.go

```diff
@@ -46,7 +48,7 @@ const (
 )
 
 type ClusterConfig struct {
-	nodeRoles            []string
+	NodesAndRoles        []nodepools.NodeRoles `json:"nodesAndRolesRKE1" yaml:"nodesAndRolesRKE1" default:"[]"`
```

Comment:
can you keep this private? Otherwise this PR is good to go!

Reply:
done